toolchain/sem_ir/value_stores.h

```diff
@@ -43,6 +43,48 @@ class NodeStore {
   ValueStore<NodeId, Node> values_;
 };
 
+// Provides a ValueStore-like interface for names. This is currently a wrapper
+// around a string store that has no state of its own.
+class NameStore {
```

Comment:
Can you add comments here explaining the difference between this and the string store?

Reply:
Done.